docs/_elements/form-controls.md

```diff
@@ -281,6 +388,49 @@ lead: Form controls allow users to enter information into a page.
       </div>
     </div>
   </fieldset>
+
+  <fieldset class="usa-fieldset-inputs">
+    <legend>Date of birth</legend>
+    <span class="usa-form-hint usa-datefield-hint" id="dobHint">For example: 04 28 1986</span>
+
```

Comment:
Date input is a little tricky due to legend default styling. Could remove the first variation in favor of a consistent typography across all label-like concepts. Without the field-set class, the default is used, with it, the legend takes on the properties of a label.